src/inmanta/server/services/compilerservice.py

```diff
@@ -63,7 +63,7 @@
 
 class CompileStateListener(object):
     @abc.abstractmethod
-    async def compile_done(self, compile: data.Compile) -> None:
+    async def compile_done(self, compile: data.Compile, notify_failed_compile: bool = False) -> None:
```

Comment:
I guess this needs to be removed, right? That information is part of the Compile object.

```suggestion
    async def compile_done(self, compile: data.Compile) -> None:
```